src/components/MAPI/utils.ts

```diff
@@ -1460,7 +1554,7 @@ function getInstallationBaseURL(providerFlavor: ProviderFlavors): URL {
   return audienceURL;
 }
 
-export function isCAPZCluster(cluster: Cluster): boolean {
+export function isCAPZAlphaCluster(cluster: Cluster): boolean {
   if (cluster.spec?.infrastructureRef?.kind !== capzv1beta1.AzureCluster) {
     return false;
   }
```

Comment:
Making a distinction here that this function tests whether a cluster is an "alpha" CAPZ cluster (created from the `v20.0.0-alpha` release and found on vintage clusters)